mustang_core/lib/src/widgets/state_consumer.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
```

Comment:
internally both `material.dart` and `widgets.dart` export same packages (https://stackoverflow.com/a/56797873). 
why are we changing `material` to `widget`?

Reply:
saying this because we use `material` everywhere

Reply:
if you did this to avoid using `material` for `pub.dev` then it won't improve the points...they only check if Flutter SDK is present. Not the code.

Reply:
Framework shjould support material and cupertino widgets, nothing to do with points